### PR TITLE
Release notes for CouchbaseNetClient 3.2.5

### PR DESCRIPTION
**Pull request by RiPont**

### CODE REVIEW

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -107,6 +107,46 @@ git pull origin master
 After you have successfully built the source, it's then just a matter of referencing the binaries (.DLL files) from your consuming project.
 _Note that you can checkout a specific tag for each release as well._
 
+== Version 3.2.5 (10 December 2021)
+
+Version 3.2.5 is the fifth release of the 3.2 series.
+
+https://packages.couchbase.com/clients/net/3.3/Couchbase-Net-Client-3.2.5.zip[Download] |
```
Comment (jeffrymorris) [inline modules/project-docs/pages/sdk-release-notes.adoc]: Need read permissions set for this asset: <Code>AccessDenied</Code>

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -107,6 +107,46 @@ git pull origin master
...
+https://docs.couchbase.com/sdk-api/couchbase-net-client-3.2.5[API Reference] |
+https://www.nuget.org/packages/CouchbaseNetClient/3.2.5[Nuget]
+
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/NCBC-2851[NCBC-2851]: TimeoutExceptions continue after rebound in Failover/Eject tests.
```
Comment (jeffrymorris) [inline modules/project-docs/pages/sdk-release-notes.adoc]: Docs team has asked that we do not simply copy the title from Jira but provide more user friendly, easy to read description. Maybe this is something we do in a separate commit though as this doc as is still adds value.

**Review by jeffrymorris — APPROVED** [review]
Approving, but please read my notes.

**Comment by osfameron** [comment]
Looks good.
I'll do some gardening on the wording ( in progress but I gotta go eat now, so will finish tomorrow)

As @jeffrymorris mentioned, the .zip link needs update before we can merge in any case!

File: modules/project-docs/pages/sdk-release-notes.adoc (hunk 1 above)
Comment (jeffrymorris) [reply modules/project-docs/pages/sdk-release-notes.adoc]: Link should be https://packages.couchbase.com/clients/net/3.2/Couchbase-Net-Client-3.2.5.zip

**Review by osfameron — APPROVED** [review]
This looks good thanks @RiPont @jeffrymorris 

I've made some additional edits, mostly based on Gerrit summaries (which generally have a better user-focused description than the original Jira)

Please do look through those, to sense check if I've introduced any errors.
